app/src/main/res/layout/fragment_help_question.xml

```diff
@@ -27,7 +27,6 @@
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:textAppearance="@style/Erouska.Caption"
-                android:autoLink="web"
```

Comment:
is it unnecessary? without this attribute won't be links and e-mails clickable, or not?

Reply:
it's handled by the markdown library